src/views/friends.tsx

```diff
@@ -1,59 +1,167 @@
-import { createResource, For, Show, type Component } from "solid-js";
+import {
+  createResource,
+  createSignal,
+  For,
+  Show,
+  type Component,
+} from "solid-js";
+import me from "~/stores/me.ts";
+import { isAndroid } from "~/utils/platform";
+
+import MdiChevronRight from "~icons/mdi/chevron-right";
+import MdiMagnify from "~icons/mdi/magnify";
+import MdiShareVariant from "~icons/mdi/share-variant";
+
 import { relationships_friends } from "../api/requests/relationships/friends/list";
-import MdiChevronRight from '~icons/mdi/chevron-right'
 
 const FriendsView: Component = () => {
   const [friends] = createResource(relationships_friends);
+  const [searchQuery, setSearchQuery] = createSignal("");
+
+  const filteredFriends = () => {
+    const query = searchQuery().toLowerCase();
+    const friendsList = friends()?.data || [];
+
+    if (!query) return friendsList;
+
+    return friendsList.filter(
+      (friend) =>
+        friend.username.toLowerCase().includes(query) ||
+        friend.fullname.toLowerCase().includes(query)
+    );
+  };
 
   return (
-    <div class="pt-[env(safe-area-inset-top)] pb-20vh mb-[env(safe-area-inset-bottom)]">
-      <header class="mb-12">
-        <nav class="flex items-center justify-end px-4 pb-2 pt-4">
-          <a href="/feed">
+    <div>
+      <header class="z-20 fixed top-0 inset-x-0 bg-gradient-to-b from-black to-transparent pt-[env(safe-area-inset-top)] max-h-[72px] h-full">
+        <nav class="flex items-center justify-between px-4 pb-2 pt-4">
+          <div class="w-8" />
+
+          <p
+            class="absolute inset-x-0 w-fit mx-auto text-2xl text-center text-white font-700"
+            role="banner"
+          >
+            StayReal.
+          </p>
+
+          <a href="/feed" aria-label="Return Feed">
             <MdiChevronRight class="text-xl" />
           </a>
         </nav>
       </header>
 
-      <Show when={friends()}
-        fallback={
-          <p class="text-center text-white/50">
-            gathering your friends...
-          </p>
-        }
-      >
-        {friends => (
+      <div class="py-16 mt-[env(safe-area-inset-top)]">
```

Comment:
```suggestion
      <div class="py-16">
```

This would result in a big margin top on iOS and Android